spec/controllers/cases/data_requests_controller_spec.rb

```diff
@@ -228,126 +257,144 @@
-        get(:send_email, params:)
-        expect(assigns(:no_email_present)).to eq(true)
-      end
-    end
-
-    context "when probation document selected" do
-      let(:template_name) { "probation" }
-
-      it "routes to the send_email branston probation page" do
-        get(:send_email, params:)
-        expect(response).to render_template(:probation_send_email)
-      end
-
-      context "with confirm probation email" do
-        let(:params) do
-          {
-            id: data_request.id,
-            case_id: data_request.case_id,
-            probation_commissioning_document_email: {
-              probation: 1,
-              email_branston_archives: "yes",
-            },
-          }
-        end
-
-        it "adds the branston probation email to recipients" do
-          post(:send_email, params:)
-          expect(response).to render_template(:send_email)
-          expect(assigns(:recipient_emails)).to include(CommissioningDocumentTemplate::Probation::BRANSTON_ARCHIVES_EMAIL)
-        end
-
-        it "updates the data_request" do
-          post(:send_email, params:)
-          expect(data_request.reload.email_branston_archives).to be_truthy
-        end
-      end
-
-      context "with decline probation email" do
-        let(:params) do
-          {
-            id: data_request.id,
-            case_id: data_request.case_id,
-            probation_commissioning_document_email: {
-              probation: 1,
-              email_branston_archives: "no",
-            },
-          }
-        end
-
-        it "doesnt add the branston probation email to recipients" do
-          post(:send_email, params:)
-          expect(response).to render_template(:send_email)
-          expect(assigns(:recipient_emails)).not_to include(CommissioningDocumentTemplate::Probation::BRANSTON_ARCHIVES_EMAIL)
-        end
-      end
-
-      context "with no options chosen" do
-        let(:params) do
-          {
-            id: data_request.id,
-            case_id: data_request.case_id,
-            probation_commissioning_document_email: {
-              probation: 1,
-            },
-          }
-        end
-
-        it "raises error message" do
-          post(:send_email, params:)
-          expect(response).to render_template(:probation_send_email)
-          expect(assigns(:email)).not_to be_valid
-        end
-      end
-    end
-
-    context "with non-probation document" do
-      let(:template_name) { "prison" }
-
-      it "routes to the send_email confirmation page" do
-        get(:send_email, params:)
-        expect(response).to render_template(:send_email)
-      end
+    it "is not implemented" do
+      expect { delete :destroy, params: { case_id: offender_sar_case.id, data_request_area_id: data_request_area.id, id: data_request.id } }
+        .to raise_error NotImplementedError, "Data request delete unavailable"
     end
   end
+
+  # describe "#send_email" do
```

Comment:
You can add an x to disable all the tests in a block without having to comment them out
```suggestion
  xdescribe "#send_email" do
```